WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -3564,27 +3566,25 @@ e_byzantium = {
 		
 		capital = 560 # Cherson
 		
-		hellenic_pagan = 600
-		bosporan_faith = 200
+		hellenic_pagan = 200
+		bosporan_faith = 600
```

Comment:
Do you know the relation between the secular and religious in the Hellenic culture and faith?